dev-support/docker/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: '3.9'
+services:
```

Comment:
We could have HS2, Metastore, DBMS in a single image; wouldn't this be much easier to use? It would also drop the need for `docker-compose`.

Reply:
OK I've seen that there is an intention to also launch HMS as standalone so ignore my previous comment.